CONTRIBUTING.md

```diff
@@ -2,6 +2,52 @@
 
 Contact us on our Discord at [https://discord.gg/YjK2JAD](https://discord.gg/YjK2JAD)
 
-Feel free to make pull requests or issues, but please note that we are currently focused on making the mod, not adding more content.
+Feel free to make pull requests or issues, but please note that we are currently focused on making the mod functional, before adding more content.
 
-People who make pull requests should be familiar with the mod licenses; if you make a PR, you agree to the license.
+## PR Guidelines
+
+* Keep PR's to one topic, if possible. This permits us to use version control to find software regressions more easily.
+* Avoid mass operations such as folder moves and restructures next to logical changes in the code.
+* Use the PR message to describe changes; use markdown check marks to show intended progress towards a goal.
```

Comment:
PR messages disappear after it's merged. We also need good commit messages.

Link to https://chris.beams.io/posts/git-commit/?

Reply:
The reason I suggest PR messages is to keep track of *progress* on a branch. It's meant to be modified and not saved in source control.

For example, I may have this:

## Variable Resistor
- [x] MNA Code
- [ ] Block
- [ ] User Interface

and by the time they PR it, those are checked off and there are commits that have messages that are useful.

Reply:
Second point actually was in a PR I was about to submit, funny enough.

Reply:
Makes sense, but I still think encouraging good commit hygiene is good.